rust/utils/run-locally/src/main.rs

```diff
@@ -417,9 +416,9 @@ fn main() -> ExitCode {
     // kathy_env_double_insertion.clone().run().join();
 
     // Send some sealevel messages before spinning up the agents, to test the backward indexing cursor
-    for _i in 0..(SOL_MESSAGES_EXPECTED / 2) {
-        initiate_solana_hyperlane_transfer(solana_path.clone(), solana_config_path.clone()).join();
-    }
+    // for _i in 0..(SOL_MESSAGES_EXPECTED / 2) {
+    //     initiate_solana_hyperlane_transfer(solana_path.clone(), solana_config_path.clone()).join();
+    // }
```

Comment:
Same as above. If we don't need solana, best to remove it. If we want to have our own things instead, I guess these could stay as placeholders, maybe add TODO comments.